src/App.js

```diff
@@ -1,6 +1,15 @@
+import { Navbar } from 'components/';
+import { useAuth } from 'contexts/';
+import { NavRoutes } from 'WebappRoutes/NavRoutes';
+
 const App = () => {
+    const { authState: { isAuth } } = useAuth();
```

Comment:
you can destructure and pass the value from context itself so that you don't have to destructure it everywhere you use useAuth

Reply:
Yes, I have done this in the new branch.